README.md

```diff
@@ -14,18 +20,24 @@ Simple React state management. Made with :heart: and ES6 Proxies.
 
 <!-- toc -->
 
-- [Introduction](#introduction)
```

Comment:
Did you change the Table of Contents by hand here? There is an `npm run build-toc` script which auto-generates it from the headers in the readme. Please run that script once instead (should we add it to some pipelines too?) If you think that the TOC could be more detailed you can tweak the `scripts/buildToc.js` file. Thx!

Reply:
Or you run the `build-toc` script but the behavior changed slightly because of the updated deps? The only thing I don't really like is how it wraps the whole TOC in an unnecessary `- [React Easy State](#react-easy-state)` point.

Reply:
It was changed automatically by prettier on saving the README file. :/

Reply:
I'll try to find a way so it won't change the TOC.